pkg/utils/release/controller.go

```diff
@@ -135,3 +118,21 @@ func (s *SuiteController) GetReleaseLink(name string, namespace string) (*v1alph
 	}
 	return releaseLink, nil
 }
+
+// CreateReleaseStrategy creates a new ReleaseStrategy using the given parameters.
+func (s *SuiteController) CreateReleaseStrategy(name, namespace, pipelineName, bundle string, policy string, releaseStrategyParams []v1alpha1.Params, serviceAccountName string) (*v1alpha1.ReleaseStrategy, error) {
```

Comment:
Unless you want to force that specific order, you can group strings together and define the type just once.
```suggestion
func (s *SuiteController) CreateReleaseStrategy(name, namespace, pipelineName, bundle, policy, serviceAccountName string, releaseStrategyParams []v1alpha1.Params) (*v1alpha1.ReleaseStrategy, error) {
```